packages/gatsby-remark-images-contentful/src/utils/index.js

```diff
@@ -41,15 +55,18 @@ const buildResponsiveSizes = async ({ metadata, imageUrl, options = {} }) => {
 
   filteredSizes.push(width)
 
-  const base64Img = await getBase64Img(`${imageUrl}?w=40`)
+  const base64Img = await getBase64Img(`${formattedImgUrl}?w=40`, reporter)
```

Comment:
Unfortunately this file is not TS typed yet, but did you check what happens when this function returns `undefined`? Before this PR it would either crash hard or return a string. After this PR it may also return `undefined`. Do consumers of this function anticipate that? Or are we going to see `src="undefined"` in our output in those cases?

Could you add a test case for this?

Reply:
I guess we should still throw if undefined is returned